Source/WebKit/UIProcess/Cocoa/SOAuthorization/SOAuthorizationSession.mm

```diff
@@ -344,7 +345,7 @@ static bool isSameOrigin(const WebCore::ResourceRequest& request, const WebCore:
         return;
     }
 
-    m_page->websiteDataStore().cookieStore().setCookies(WTFMove(cookies), [weakThis = ThreadSafeWeakPtr { *this }, response = WTFMove(response), data = adoptNS([[NSData alloc] initWithData:data])] () mutable {
+    m_page->websiteDataStore().protectedCookieStore()->setCookies(WTFMove(cookies), [weakThis = ThreadSafeWeakPtr { *this }, response = WTFMove(response), data = adoptNS([[NSData alloc] initWithData:data])] () mutable {
```

Comment:
No need to put `m_page` into a local `RefPtr`?

Reply:
`WebPageProxy::websiteDataStore()` is a trivial inline function so I doubt it. In any case, the Safer CPP bot will tell me if I missed something. I'll wait for that bot before landing.